sdk/cognitivelanguage/ai-language-text/src/lro.ts

```diff
@@ -47,19 +47,36 @@ const jobStatusOperationSpec: OperationSpec = {
   serializer,
 };
 
+function addOnResponse<TOptions extends OperationOptions>(
+  options: TOptions,
+  cb: (rawResponse: FullOperationResponse, response: unknown, error: unknown) => void
+): TOptions {
+  return {
+    ...options,
+    onResponse: (rawResponse, response, error) => {
+      cb(rawResponse, response, error);
+      options.onResponse?.(rawResponse, response, error);
+    },
+  };
+}
+
+function logWanrHeader(rawResponse: FullOperationResponse) {
+  const warnHeader = rawResponse.headers.get("warn-text");
```

Comment:
I checked my code and I'm logging: "warn-code", "warn-agent", "warn-text". Is that overkill? Is all the information in the warn-text?

Reply:
I believe from the customer point of view, they're only interested in warn-text. I can't think of a reason why they would be interested in warn-code in a log message. Are they even documented somewhere and can be acted upon in a meaningful way? We can log it later if it turns out it is useful.

I am not sure what warn-agent is, this is a screenshot you took earlier for a response with these headers: 
![image](https://user-images.githubusercontent.com/6074665/220818446-38fe4b10-f3bc-489c-9376-01220806b304.png)

Reply:
Thank you for finding that screenshot. Yeah, it looks like warn-text is really only relevant.